vispy/visuals/markers.py

```diff
@@ -474,7 +474,6 @@
     '^': triangle_up,
     'v': triangle_down,
     '*': star,
-    None: None,
 }
 marker_types = tuple(sorted(list(_marker_dict.keys())))
 
```

Comment:
@campagnola Why would you even instantiate a marker class if you're not going to set a marker? The only thing I can think of is to "disable" it for a while, but with the new SceneGraph you can turn node visibility off and on, right?